find-plugin-repos.sh

```
@@ -39,27 +39,31 @@ source jenkinsfile_check.sh
-  # Check if the curl command was successful
-  if [[ "$jenkinsfile" != "404: Not Found"* ]]; then
+  # Check the HTTP status code
+  if [ "$http_code" -eq 200 ]; then
     info "Jenkinsfile found in $repo"
     # Check if the Java version numbers exist in the Jenkinsfile
     check_java_version_in_jenkinsfile "$jenkinsfile" "$repo"
-  else
+  elif [ "$http_code" -eq 404 ]; then
     # Format the repository name
     formatted_repo=$(format_repo_name "$repo")
     echo "$formatted_repo,https://github.com/$repo" >>"$csv_file_no_jenkinsfile"
```

Shellcheck (0.10.0)

[warning] csv_file_no_jenkinsfile is referenced but not assigned.

(SC2154)